core-common/src/main/resources/kylin-defaults.properties

```diff
@@ -336,6 +336,7 @@ kylin.engine.spark-conf-mergedict.spark.memory.fraction=0.2
 kylin.engine.flink-conf.jobmanager.heap.size=2G
 kylin.engine.flink-conf.taskmanager.heap.size=4G
 kylin.engine.flink-conf.taskmanager.numberOfTaskSlots=1
+kylin.engine.flink-conf.env.hadoopclasspath=/usr/hdp/2.4.0.0-169/hadoop/
```

Comment:
Can this Hadoop path be detected at runtime? It is difficult for the user to configure this. Please refer to the "find-hadoop-conf-dir.sh"

Reply:
OK, I will rethink this.